### PR TITLE
Replace ActiveJDBC with JDBI for QueryHistory

### PR DESCRIPTION
**Pull request by ebyhr**

### CODE REVIEW

**Comment by ebyhr** [comment]
@mosabua JDBI is a de facto standard in Trino project.

**Comment by mosabua** [comment]
You are still working on this right @ebyhr ? Let me know when you want me to look...

File: gateway-ha/pom.xml [hunk 1]
```diff
@@ -262,6 +262,18 @@
             <version>${dep.activejdbc.version}</version>
```
Comment (mosabua) [inline gateway-ha/pom.xml:262]: Remove this and any traces of it?

Comment (ebyhr) [reply gateway-ha/pom.xml:262]: I don't understand your suggestion? We can't remove the line or the dependency as it's still used.

File: pom.xml [hunk 2]
```diff
@@ -58,6 +58,7 @@
         <dep.h2.version>1.4.192</dep.h2.version>
         <dep.jakarta.annotations.version>2.1.1</dep.jakarta.annotations.version>
         <dep.jakarta.mail.version>2.0.0</dep.jakarta.mail.version>
+        <dep.jdbi.version>3.42.0</dep.jdbi.version>
```
Comment (wendigo) [inline pom.xml:61]: nit: 3.44.0

**Review by wendigo — APPROVED** [review]

File: gateway-ha/pom.xml (hunk 1 above)
Comment (mosabua) [reply gateway-ha/pom.xml:262]: I misdunerstood the scope of the PR .. i just glanced and thought this removes all usage of activejdbc .. but that is not the case.. not yet ;-)

**Pull request merged**